desruptionPosX: -0.5
desruptionPosY: 1.8
toqueZNm: 50.0
forceXN: 200.0
duration: 0.5
